apps/web/src/components/icons/pdf-icon.tsx

```
@@ -0,0 +1,57 @@
+	const style: CSSProperties | undefined = size ? { width: size, height: size } : undefined;
+
+	return (
+		<svg
+			viewBox="0 0 400 400"
+			xmlns="http://www.w3.org/2000/svg"
+			fill={color}
+			stroke={color}
+			className={className}
+			style={style}
+		>
+			<g id="SVGRepo_bgCarrier" strokeWidth="0"></g>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)